docker-compose-legal.yml

```diff
@@ -0,0 +1,21 @@
+version: '2.1'
+
+services:
+  zap-proxy:
```

Comment:
this looks like it should be in a shared docker-compose file, I can't see any changes here.

Reply:
We probably need shared compose file for T1 and T2 in general, but that's a bigger task. Maybe we could leave this as is and get rid of all such duplications at once?

Reply:
I think for the work required not to duplicate this we should do this now

Reply:
Fine with me 👌